ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -85,6 +85,7 @@
       src: /tmp/oc
       dest: "{{ oc_tools_dir }}/oc"
       mode: 0755
+    when: local_oc_client is defined and local_oc_client
```

Comment:
When we talked I understood this change's goal to be _install oc client when not `local_oc_client`_, this looks backwards to me.